src/workerd/api/web-socket.h

```diff
@@ -407,11 +410,13 @@ class WebSocket: public EventTarget {
       JSG_READONLY_PROTOTYPE_PROPERTY(url, getUrl);
       JSG_READONLY_PROTOTYPE_PROPERTY(protocol, getProtocol);
       JSG_READONLY_PROTOTYPE_PROPERTY(extensions, getExtensions);
+      JSG_READONLY_PROTOTYPE_PROPERTY(autoResponseTimestamp, getAutoResponseTimestamp);
```

Comment:
We do need to be careful about adding more non-standard properties to the WebSocket api. Every non-standard method and property we add here runs a non-trivial risk of conflicts and incompatibilities with the standard, and backwards compat headaches later.